packages/input-amount/src/LionInputAmount.js

```diff
@@ -53,14 +53,11 @@ export class LionInputAmount extends LocalizeMixin(LionInput) {
   }
 
   static get styles() {
-    return [
-      super.styles,
-      css`
-        .input-group__container > .input-group__input ::slotted(.form-control) {
-          text-align: right;
-        }
-      `,
-    ];
+    return css`
```

Comment:
Forgot why the parent does not have any styles. Everything inheriting from FormControlMixin should add `super.styles` right?